rasa_nlu/extractors/duckling_extractor.py

```diff
@@ -136,6 +136,7 @@ def process(self, message, **kwargs):
                       "end": match["end"],
                       "text": match["text"],
                       "value": value,
+                      "confidence": 1.0,
```

Comment:
does this need to be added to duckling_http_extractor?

Reply:
yes!